app/build.gradle

```diff
@@ -11,15 +11,15 @@ spotbugs {
 }
 
 android {
-    compileSdkVersion 30
-    buildToolsVersion "30.0.3"
+    compileSdkVersion 31
+    buildToolsVersion "31.0.0"
 
     defaultConfig {
         applicationId "me.hackerchick.catima"
         minSdkVersion 19
-        targetSdkVersion 30
-        versionCode 83
-        versionName "2.6.0"
+        targetSdkVersion 31
+        versionCode 84
+        versionName "2.6.1"
```

Comment:
Please don't update versionCode and versionName, I like to do this myself right when I release because at that point I am also most sure if this will be a new major, minor or patch.